Ubuntu v16.04 and older does not automatically generate sssd.conf

jessawong opened the pull request:
The original instructions work only for for Ubuntu 17+ as posted here: https://bugs.launchpad.net/ubuntu/+source/sssd/+bug/1691726

You are required to make your own conf file before installing the sssd dependencies, otherwise sssd will fail on installation.

azurecla commented:
Hi __@AndrewHoh__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (anhoh). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

rmca14 commented:
@barbkess

rmca14 commented:
@barbkess 
Please disregard, thanks.

jessawong committed:
Ubuntu v16.04 and older does not automatically generate sssd.conf

The original instructions work only for for Ubuntu 17+ as posted here: https://bugs.launchpad.net/ubuntu/+source/sssd/+bug/1691726

You are required to make your own conf file before installing the sssd dependencies, otherwise sssd will fail on installation.

mahesh-unnikrishnan reviewed (changes requested):
Thanks for sharing this feedback. I would suggest moving the new section (Additional Step for Ubuntu v16.04) just above the section titled '
Join the Linux virtual machine to the managed domain'.

Please let me know when you've made this change & we can proceed to merge it.
